diff --git a/node_modules/json-bigint/lib/stringify.js b/node_modules/json-bigint/lib/stringify.js
index 3bd5269..c13fe3a 100644
--- a/node_modules/json-bigint/lib/stringify.js
+++ b/node_modules/json-bigint/lib/stringify.js
@@ -215,7 +215,7 @@ var JSON = module.exports;
             mind = gap,
             partial,
             value = holder[key],
-            isBigNumber = value != null && (value instanceof BigNumber || BigNumber.isBigNumber(value));
+            isBigNumber = false;
 
 // If the value has a toJSON method, call it to obtain a replacement value.
 
